.make.defaults

```diff
@@ -219,3 +219,32 @@ __check_defined = \
 	pip install wheel;			\
 	pip install -r requirements.txt;	\
 
+.PHONY: .defaults.check.installed
+.defaults.check.installed::
+	@installed=$$(which $(CHECK_RUNNABLE));		\
+	if [ -z "$$installed" ]; then			\
+	    echo $(CHECK_RUNNABLE) must be installed;	\
+	    exit 1;					\
+	fi
+
+.PHONY: .defaults.minio.check
+.defaults.minio.check::
+	@# Help: Check that minio and the mc CLI is installed. 
+	$(MAKE) CHECK_RUNNABLE=minio .defaults.check.installed
+	$(MAKE) CHECK_RUNNABLE=mc .defaults.check.installed
+
+.PHONY: .defaults.minio.start
+.defaults.minio.start::
+	$(MAKE) .defaults.minio.check
+	@# Help: Start the minio server and alias to local
+	minio server start &
+	mc admin info local
+	mc alias set local http://127.0.0.1:9000 minioadmin minioadmin
+
+.PHONY: .defaults.minio.load-test-data
```

Comment:
You are missing creation of minio secrets

Reply:
Fixed